corehq/apps/app_manager/templates/app_manager/default_followup_form.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>{{ name }}</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/6E2725BB-7A7B-4CFD-A27D-BA7CD4D790DB" uiVersion="1" version="1" name="Followup Form">
+					<default_label />
+				</data>
+			</instance>
+			<bind vellum:nodeset="#form/default_label" nodeset="/data/default_label" />
+			<itext>
+				<translation lang="{{ lang }}" default="">
+					<text id="default_label-label">
+						<value> {{ default_label }} </value>
```

Comment:
remove the outer spaces - as is, this will put an unexpected leading and trailing space in the label